scriptmodules/supplementary/skyscraper.sh

```diff
@@ -469,89 +419,85 @@ function gui_skyscraper() {
     local -a s_source_names
     local -A help_strings
 
-    s_source=([1]=screenscraper [2]=arcadedb [3]=thegamesdb [4]=igdb [5]=openretro [6]=worldofspectrum)
-    s_source+=([10]=localdb [11]=import)
+    s_source=([1]=screenscraper [2]=arcadedb [3]=thegamesdb [4]=openretro [5]=worldofspectrum)
+    s_source+=([10]=esgamelist [11]=import)
 
-    s_source_names=([1]=ScreenScraper [2]=Arcade Database [3]=TheGamesDB [4]=IGDB [5]="OpenRetro" [6]="World of Spectrum")
-    s_source_names+=([10]="LocalDB Cache" [11]="Import Folder")
+    s_source_names=([1]=ScreenScraper [2]=ArcadeDB [3]=TheGamesDB [4]="OpenRetro" [5]="World of Spectrum")
+    s_source_names+=([10]="Emulationstation Gamelist" [11]="Import Folder")
 
```

Comment:
these would be more readable if not done as long lines (and then no += is needed).

```
array=(
    [1] = "Blah"
...
```

etc

Reply:
(as you have done elsewhere anyway) with the help strings.

Reply:
Sure, no problem.